apps/webapp/src/fetchers/transactions.ts

```diff
@@ -1,6 +1,6 @@
 import { viewClient } from '../clients/grpc';
 import { streamToPromise } from './stream';
-import { classifyTransaction, uint8ArrayToHex } from '@penumbra-zone/types';
+import { getTransactionClassificationLabel, uint8ArrayToHex } from '@penumbra-zone/types';
```

Comment:
This is actually a new function that does the exact same thing as the original. But I repurposed the original to return a machine-readable value rather than a human-readable value.

Reply:
what's the difference between machine readable and human readable?

Reply:
The machine-readable classifications are defined by [the `TransactionClassification` type](https://github.com/penumbra-zone/web/pull/387/files/0e4575541dc4bfdffc5978872139ac35d3bb3eee#diff-fdf698291d45d9c5719ccb4b9ebcdd6e0e15d4d71e73c8615e6fe1da1bf068fa), and can be used with e.g., `switch` or `if` statements to determine what kind of UI to render. Currently, [it's used inside `ViewTabs`](https://github.com/penumbra-zone/web/pull/387/files/0e4575541dc4bfdffc5978872139ac35d3bb3eee#diff-cbbb8cbe51aa9dff111ee1ae2c9e8036e43a3874cbd5c6c7cee6d08222f4e26aR16-R17) to determine which tabs to render, based on whether a transaction is a `send` vs. some other type.

Reply:
Oops, I posted the comment before finishing.

The human-readable value is a label that shows up in the UI. So, while machine-readable values are defined in [the `TransactionClassification` type](https://github.com/penumbra-zone/web/pull/387/files/0e4575541dc4bfdffc5978872139ac35d3bb3eee#diff-fdf698291d45d9c5719ccb4b9ebcdd6e0e15d4d71e73c8615e6fe1da1bf068fa), the human-readable values are defined by [the function](https://github.com/penumbra-zone/web/pull/387/files/0e4575541dc4bfdffc5978872139ac35d3bb3eee#diff-4dc78f0802f529efed129c435d57ea2f67cab9853483548ba25752882fdff965R70-R79) that returns the label.